tests/inline.js

```diff
@@ -168,15 +170,33 @@ describe( 'InlineEditor', () => {
 			return InlineEditor.create( editorElement, { plugins: [ Paragraph ] } )
 				.then( newEditor => {
 					editor = newEditor;
+
+					const schema = editor.document.schema;
+
+					schema.registerItem( 'bold' );
+					schema.allow( { name: 'bold', inside: '$root' } );
+					schema.allow( { name: '$text', inside: 'bold' } );
+
+					buildViewConverter().for( editor.data.viewToModel )
+						.fromElement( 'b' )
+						.toElement( 'bold' );
+
+					buildModelConverter().for( editor.data.modelToView )
+						.fromElement( 'bold' )
+						.toElement( 'b' );
+
+					buildModelConverter().for( editor.editing.modelToView )
```

Comment:
Yy... Did you copy it from somewhere? :D What is this converter for in this test?

I think that it'd be better to just load some real features in this test. Basic styles and paragraph features are dev deps of this package anyway.